benchmark_runner/common/clouds/BareMetal/bare_metal_operations.py

```diff
@@ -239,6 +239,7 @@ def run_ocp_upgrade(self, oc: OC):
         """
         if f'Cluster version is {self._upgrade_ocp_version}' == oc.get_cluster_status():
             logger.info(f'Cluster is already upgraded to: {self._upgrade_ocp_version}')
+            return 'Cluster is already upgraded'
```

Comment:
Should return True, not a string (or you need to change the comment above)..  Also note that there's no return if the version is not correct (existing code, but it looks wrong).